runtime/interface.cpp

```diff
@@ -2345,6 +2368,8 @@ static void init_runtime_libs() {
   init_string_buffer_lib(static_cast<int>(static_buffer_length_limit));
 
   init_interface_lib();
+
+  vk::singleton<ThreadPool>::get().init();
```

Comment:
let's move it to `worker_global_init`
https://github.com/VKCOM/kphp/blob/a9aafd0bf159fa43675253fe336ddebad2b4943b/runtime/interface.h#L209

Reply:
done